src/ape/managers/project.py

```diff
@@ -680,11 +689,11 @@ def run_script(self, name: str, interactive: bool = False):
             return console()
 
     def _load_dependencies(self) -> Dict[str, PackageManifest]:
-        return {d.name: d.extract_manifest() for d in self.config.dependencies}
+        return {d.name: d.extract_manifest() for d in self.config_manager.dependencies}
 
     # @property
```

Comment:
ugh commented out code? Can we not?

Reply:
Commented out code is the antithesis of clean code.
It sits here rots, no one knows why it is here, everyone is afraid to delete it, it looks bad, it is hard to maintain...

I would strongly prefer to keep "ideas" out of the production code base, especially as we near  v1.  Keep "ideas" in GH issues